src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/flyout/flyout.js

```diff
@@ -629,16 +755,29 @@ class FlyoutUI extends Component {
 
   renderSubheader() {
     if (
-      !this.hasConflicts ||
-      this.state.isLoading ||
-      this.state.wasImportSuccessful
+      this.state.status === 'loading' ||
+      this.state.status === 'success'
     ) {
       return null;
     }
 
-    return (
-      <Fragment>
-        <EuiSpacer size="s" />
+    let legacyFileWarning;
+    if (this.state.isLegacyFile) {
+      legacyFileWarning = (<EuiCallOut
+        title={(
+          <FormattedMessage
+            id="kbn.management.objects.objectsTable.flyout.legacyFileUsedTitle"
+            defaultMessage="Importing .json files is deprecated"
```

Comment:
I think message should give some more guidance on how to fix this problem, preferably with a link to documentation.

Reply:
@gchaps any suggestions here? We need a message to indicate to the user that in a future version their files from the previous export (`.json` files) will no longer be supported. Also that they should generate a new export (`.ndjson` files) using the new export before it's too late.

Reply:
Will either of these work?

**Importing JSON files is no longer supported**

We now support NDJSON files, which you can generate using our updated export.

**Support for JSON files is going away**

Use our updated export to generate NDJSON files, and you'll be all set.

Reply:
I like the second option, I have attached a screenshot of the updated UI.

<img width="403" alt="Screen Shot 2019-04-08 at 9 05 26 PM" src="https://user-images.githubusercontent.com/3694571/55766493-e6eace00-5a42-11e9-9d66-671d080f92ac.png">